src/interfaces/IParaSwapRepayAdapter.sol

```diff
@@ -22,19 +18,18 @@ interface IParaSwapRepayAdapter is IBaseParaSwapAdapter {
     uint256 debtRepayAmount; // the amount of debt to repay
     uint256 debtRepayMode; // debt interest rate mode (1 for stable, 2 for variable)
     uint256 offset; // offset in buy calldata in case of swapping all collateral, otherwise 0
+    uint256 flashLoanAmount; // the amount to flashloan equivalent to the debt to repay
```

Comment:
there is no `debtRepayAmount` already?

Reply:
`flashloanAmount` will not be equal to `debtRepayAmount`. Flashloan will be of collateral asset which will be swapped to debt asset. Both will be different.

Reply:
that's not `maxCollateralAmountToSwap` then?